src/pages/using-spark/foundations/design-tokens.mdx

```diff
@@ -0,0 +1,76 @@
+---
+  title: Design Tokens
+---
+import SassVarTable from '../../../components/SassVarTable';
+
+<h1 class="docs-b-h1 sprk-b-PageTitle sprk-b-TypeDisplayOne sprk-u-mbl sprk-u-Measure">
+  Design Tokens
+</h1>
+
+Our Design Tokens are a catalog from which we base all our visual and branding choices.
+It's a central database of our core style attributes.
+
+These tokens describe branding decisions in color, typography, spacing, sizing, and much more.
+
+Design Tokens are a future-friendly way to create visual consistency between platforms.
+
+## Benefits
+Design Tokens allow more projects to incorporate our design decisions.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+  Designer Perks
+</h3>
+
+- Encourages visual consistency and design consensus across products.
+- Creates easier syncing with brand guidelines.
+- Encourages better support for multi-platform projects.
+- Promotes clear communication with developers and other designers.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+  Developer Perks
+</h3>
+
+- Maintains sync with current style updates while minimizing impact to code.
+- Enables Android and iOS projects to access style variables from Spark.
+- Supports future-friendly development. Design Tokens can generate style files for any possible platform in the future.
+
+## How to Use Our Design Tokens
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mbm sprk-u-Measure">
+ For Designers
+</h3>
+
+Use token names to communicate the specific style values of your design.
+
+Token names depict style choices that portray color, spacing, typography, and more.
+
+For example, the token `sprk-purple-deep` represents a specific shade of purple. Style consensus makes it clear to everyone which purple you mean.
+
+<h3 class="docs-b-h5 sprk-b-TypeDisplayFive sprk-u-mtl sprk-u-mbm sprk-u-Measure">
+ For Developers
+</h3>
+
+For HTML, React, and Angular projects, follow our [install guides](/installing-spark). They show you how to import the correct Sass file.
+
+<h4 class="docs-b-h6 sprk-b-TypeDisplaySix sprk-u-mtl sprk-u-mbm sprk-u-Measure">
+  How to Change Spark's Default Style Settings:
```

Comment:
We might want to make this clear that these instructions are for changing it in Web (Sass).

Reply:
I'll take care of this!

Reply:
DONE!
https://github.com/sparkdesignsystem/spark-design-system/pull/3663/files#diff-ea2f00dc3ed9140568adb2a57fbcb355ade147c3b4e9b7bd6e3d9754ecd79986